crates/re_log_types/src/time.rs

```diff
@@ -1,5 +1,16 @@
 use std::ops::RangeInclusive;
-use time::{OffsetDateTime, UtcOffset};
+use time::{format_description::FormatItem, OffsetDateTime, UtcOffset};
+
+// TODO(paris): Added `serde::Deserialize, serde::Serialize` to fix error in `crates/re_viewer_context/src/app_options.rs`:
```

Comment:
makes sense, that is indeed needed for having it in the app options sine app options is serialized with serde, so everything it contains needs to be serializable as well. Your solution is correct

Reply:
Sounds good, thanks.